ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.view.js

```diff
@@ -104,9 +117,13 @@ module.exports = Marionette.LayoutView.extend({
         user.savePreferences();
         this.triggerCloseDropdown();
     },
+    triggerRender() {
```

Comment:
No need for this method since you can just call this.render directly.

Reply:
Done